src/routers/alpha-router/alpha-router.ts

```diff
@@ -1178,42 +1178,12 @@ export class AlphaRouter
       cacheMode !== CacheMode.Darkmode &&
       swapRouteFromChain
     ) {
-      const tokenPropertiesMap = await this.tokenPropertiesProvider.getTokensProperties([tokenIn, tokenOut], providerConfig);
```

Comment:
FYI, We also no longer need to call `tokenPropertiesProvider.getTokensProperties` for `cachedRoutes.tokenIn` and `cachedRoutes.tokenOut`, because `swapRouteFromChain.routes` will contain the pool reserve with FOT tax, and can apply to `QuoteProvider.getQuotes` correctly. (i.e.,  `cachedRoutes.tokenIn` and `cachedRoutes.tokenOut` FOT tax will get discarded, and Pair.reserve FOT tax will be reapplied in `QuoteProvider.getQuotes`